test/lms_test.c

```diff
@@ -571,6 +572,11 @@ int setup_tests(void)
     if (!test_get_libctx(&libctx, &nullprov, config_file, &libprov, NULL))
         return 0;
 
+    ctx = EVP_PKEY_CTX_new_from_name(libctx, "LMS", propq);
+    if (ctx == NULL && ERR_get_error() == EVP_R_UNSUPPORTED_ALGORITHM)
+        return TEST_skip("LMS algorithm is not available in provider");
```

Comment:
@paulidale Is this right here?
It looks like cleanup_tests() only gets called when setup_tests() succeeds.
Which means it manually needs to be cleaned up here?